### DISCUSSION TITLE
Deleting the rule does not stop the alert

### DISCUSSION THREAD

**Discussion by hyperbaba** [jertel/elastalert2#113] [Q&A] [answered]
When i delete a rule it does not stop the alerting from that rule . The only way is to restart a elastalert

I think that was the old problem but was eventually solved on elastalert tree

**Comment by nsano-rururu** [comment]
It's an existing move.

**Comment by nsano-rururu** [comment]
If you know how to fix it, please make a pull request.

If you can't, please provide information if you have anything related to a yelp / elastalert issue or pull request.

**Comment by nsano-rururu** [comment]
I'm importing the content of the following pull request, does it still occur?
[Fix is_enabled not work with reload #3036](https://github.com/Yelp/elastalert/pull/3036)

**Comment by nsano-rururu** [comment]
What is "elastalert tree"?

**Comment by nsano-rururu** [comment]
@hyperbaba

If the issue has already been resolved, please do not post it in the issue. I beg you.

**Comment by jertel** [comment, marked as answer]
I've updated the documentation to make the procedure for disabling and deleting rules more clear. https://elastalert2.readthedocs.io/en/latest/running_elastalert.html#disabling-a-rule

I've also moved this into a discussion since the guidelines for creating an issue were not met.

**Reply by nsano-rururu** [reply]
@jertel 

I didn't know it was in the documentation. Thank you for the information.

**Comment by hyperbaba** [comment]
Thank you for the answers, it helped alot. The 5 min delay was the reason for me to raise the alarm :) Documentation explains it now.

Thank You,
